### PR TITLE
Added fix when constructing default base and vocab

### PR DESCRIPTION
**Pull request by umbreak**

### CODE REVIEW

File: src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala [hunk 1]
```diff
@@ -46,6 +45,9 @@ class Projects[F[_]](agg: Agg[F], index: ProjectCache[F], organizations: Organiz
     retryStrategy: RetryStrategy[F]
 ) {
 
+  private def invalidIriGeneration(organization: String, label: String, param: String): String =
+    s"the value of the project's '$param' could not be generated properly from the provided project '$organization/$label'"
```
Comment (wwajerowicz) [inline src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala]: Is this even possible given the fact that the `$organization/$label` is part of the URL by which project is accesed?

File: src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala [hunk 2]
```diff
@@ -63,19 +65,18 @@ class Projects[F[_]](agg: Agg[F], index: ProjectCache[F], organizations: Organiz
...
-                                        org.uuid,
-                                        organization,
-                                        project.description,
-                                        project.apiMappings,
-                                        base,
-                                        vocab,
-                                        clock.instant,
-                                        caller)
-            evaluateAndUpdateIndex(command) <* retryStrategy(setOwnerPermissions(organization, label, caller))
+            project.baseOrGenerated(organization, label) -> project.vocabOrGenerated(organization, label) match {
```
Comment (wwajerowicz) [inline src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala]: Why use `->` here? it's not a map so it's a bit confusing, perhaps tuple would be more clear.

File: src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala (hunk 1 above)
Comment (bogdanromanx) [reply src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala]: thats a good point...

File: src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala (hunk 2 above)
Comment (wwajerowicz) [reply src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala]: it is, but for me `->` usually means that there's some sort of mapping between elements :)

**Pull request merged**